samples/ControlCatalog.NetCore/Program.cs

```diff
@@ -4,29 +4,43 @@
 using System.Runtime.InteropServices;
 using System.Threading;
 using Avalonia;
-using Avalonia.Controls;
 
 namespace ControlCatalog.NetCore
 {
     static class Program
     {
         static void Main(string[] args)
         {
+            if (args.Contains("--wait-for-attach"))
+            {
+                Console.WriteLine("Attach debugger and use 'Set next statement'");
+                while (true)
+                {
+                    Thread.Sleep(100);
+                    if (Debugger.IsAttached)
```

Comment:
Couldn't you just use the `Debugger.Launch` method here instead since that will automatically prompt the user to attach a debugger?

Reply:
It doesn't work on Linux/OSX.

Reply:
Makes sense.